### PR TITLE
Add link to Keymap page in book.

### PR DESCRIPTION
**Pull request by python128**
This updates the remapping page to open the Keymap page of book, rather than the commands file.

I have also updated the original link to open at line #L178(opens at the invocation of `commands!` macro)

### CODE REVIEW

File: book/src/remapping.md [hunk 1]
```diff
@@ -53,4 +53,5 @@ Control, Shift and Alt modifiers are encoded respectively with the prefixes
 
 Keys can be disabled by binding them to the `no_op` command.
 
-Commands can be found in the source code at [`helix-term/src/commands.rs`](https://github.com/helix-editor/helix/blob/master/helix-term/src/commands.rs)
+Commands can be found at [Keymap](https://docs.helix-editor.com/keymap.html) Commands.(Right pane)
```
Comment (sudormrfbin) [inline book/src/remapping.md:56]: `Commands.(Right pane)` at the end ?

File: book/src/remapping.md [hunk 2]
```diff
@@ -53,4 +53,5 @@ Control, Shift and Alt modifiers are encoded respectively with the prefixes
...
+> Commands can also be found in the source code at [`helix-term/src/commands.rs`](https://github.com/helix-editor/helix/blob/master/helix-term/src/commands.rs#L178)
```
Comment (sudormrfbin) [inline book/src/remapping.md:57]: It's better to not have the line number since it will become outdated pretty easily.

Comment (sudormrfbin) [inline book/src/remapping.md:57]: It's better to not have the line number since it will become outdated pretty easily.

Comment (python128) [reply book/src/remapping.md:57]: Ah, ok

File: book/src/remapping.md (hunk 1 above)
Comment (python128) [reply book/src/remapping.md:56]: I'm sorry, didn't get you.

I put it there just to make sure the people use the `Commands` pane, which is the right side.

Comment (sudormrfbin) [reply book/src/remapping.md:56]: Ah you meant the right column, I don't think we need to mention it since it's pretty self evident.

Comment (python128) [reply book/src/remapping.md:56]: 👍 I'll remove it.

Comment (python128) [reply book/src/remapping.md]: Done 👍

**Review by sudormrfbin — APPROVED** [review]

**Pull request merged**